manifest.konnector

```diff
@@ -1,18 +1,15 @@
 {
   "version": "1.0.0",
-  "name": "konnector name",
-  "type": "node",
-  "slug": "konnectorslug",
-  "description": "description",
-  "source": "git://github.com/konnectors/cozy-konnector-thename.git",
+  "name": "Boulanger",
+  "type": "io.cozy.bills",
```

Comment:
You must keep the `node` type, in case we authorize some connectors in other languages later

Reply:
Oh ! Done